tests/unit/util/perfc_test.c

```diff
@@ -114,6 +114,7 @@ MTF_DEFINE_UTEST(perfc, perfc_get_cycles)
 
     free(cyclev);
 }
+#endif
```

Comment:
Probably HSE_CI or something longer and more distinguisable would be better in the long run...

Reply:
You are right. Will change.